hpimdm/Run.py

```diff
@@ -64,40 +65,52 @@ def run(self):
                 print(sys.stderr, 'sending data back to the client')
                 print(pickle.loads(data))
                 args = pickle.loads(data)
+                if 'ipv4' not in args and 'ipv6' not in args or not (args.ipv4 or args.ipv6):
```

Comment:
This should be `'ipv4' not in args or 'ipv6' not in args or not (args.ipv4 or args.ipv6):`